internal/topic/topiclistenerinternal/stream_listener.go

```diff
@@ -187,7 +187,7 @@ func (l *streamListener) initVars(sessionIDCounter *atomic.Int64) {
 	}
 }
 
-//nolint:funlen
+//nolint:funlen,nolintlint
```

Comment:
```suggestion
//nolint:funlen
```

No issues with `nolint` here